packages/google_sign_in/google_sign_in/pubspec.yaml

```diff
@@ -16,27 +16,21 @@ flutter:
         default_package: google_sign_in_web
 
 dependencies:
-  google_sign_in_platform_interface: ^1.1.1
+  google_sign_in_platform_interface: ^2.0.0-nullsafety
   flutter:
     sdk: flutter
-  meta: ^1.0.4
-  # The design on https://flutter.dev/go/federated-plugins was to leave
-  # this constraint as "any". We cannot do it right now as it fails pub publish
-  # validation, so we set a ^ constraint.
-  # TODO(amirh): Revisit this (either update this part in the  design or the pub tool).
-  # https://github.com/flutter/flutter/issues/46264
-  google_sign_in_web: ^0.9.1
```

Comment:
This pubspec is declaring a 'default_package: google_sign_in_web' for the 'web' platform (line 16), however it's removing the dependency on the web package here. I don't think this'll work for web users. This should be documented in the CHANGELOG.md entry.

Reply:
We had discussed this somewhere in this PR and decided to remove the default package. I am not familiar with how that needs to be executed upstream. Could you please help with that David?

Reply:
I'm not really sure if this is used on-demand or validated somehow; I think the web endorsement should be removed (otherwise we're cheating pub.dev into reporting "web" support for the null-safe version, which is not true)

Reply:
Tracking here: https://github.com/flutter/flutter/issues/72239